src/applications/bmqstoragetool/m_bmqstoragetool_printer.cpp

```diff
@@ -641,9 +641,9 @@ class JsonPrinter : public Printer {
         BSLS_KEYWORD_OVERRIDE
     {
         closeBraceIfOpen();
-        d_ostream << "  \"OutstandingRatio\": \"" << ratio
-                  << "\",\n  \"OutstandingMessages\": \""
-                  << outstandingMessagesCount << "\"";
+        d_ostream << "  \"OutstandingRatio\": " << ratio
```

Comment:
It seems it is discarded change from PR https://github.com/bloomberg/blazingmq/pull/665.